src/collection-browser.ts

```diff
@@ -936,16 +947,16 @@ export class CollectionBrowser
           size: 50,
         },
       ],
-    };
+    }; */
 
     const params: SearchParams = {
       query: this.fullQuery,
-      fields: ['identifier'],
-      aggregations,
-      rows: 1,
+      // aggregations, // get default aggregations
```

Comment:
this one as well, better to add note here as well for more context

Reply:
👍 Contextual note added in latest commit